src/derivative_operators/BC_operators.jl

```diff
@@ -1,32 +1,35 @@
 abstract type AbstractBC{T} <: AbstractDiffEqLinearOperator{T} end
```

Comment:
We may want to get rid of this, in general our BC operators are affine as opposed to linear.

Reply:
I don't remember adding it personally, happy to remove it. I think that AbstractBC should hook in to some unified type along with AbstractDerivativeOperator etc. somewhere to allow for later extension